internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -295,19 +296,24 @@ ChunkedSegmentSealedImpl::load_column_group_data_internal(
                                   schema_->ShouldLoadField(milvus_field_ids[i]);
         }
 
+        auto mmap_dir_path =
```

Comment:
this handles only v2, but not v1. also are we deprecating `load_info.mmap_dir_path`? if so we should try to remove that field from load_info